<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SJF">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SJF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FCFS">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>FCFS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RoundRobin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Round Robin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="priority">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>priority</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Preemptivepriority">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Preemptive priority </string>
    </property>
   </widget>
   <widget class="QLabel" name="lab">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>89</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>number of processes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cpu scheduling </string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitdata">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>500</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="processes">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="PreemptiveSJF">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Preemptive SJF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
